cmd/client.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/minio/mc/pkg/probe"
 	minio "github.com/minio/minio-go/v6"
 	"github.com/minio/minio-go/v6/pkg/encrypt"
+	tagging "github.com/minio/minio/pkg/bucket/object/tagging"
```

Comment:
```suggestion
	"github.com/minio/minio/pkg/bucket/object/tagging"
```

Reply:
This insertion is done by VSCode by default.